sgkit/tests/test_utils.py

```diff
@@ -45,3 +45,39 @@ def check(x: ArrayLike, values: ArrayLike, names: List[Any]) -> None:
         [0, 0, 1, 2, 1, 3, 3, 1],
         [2.0, 1.0, 3.0, 5.0],
     )
+
+
+@pytest.mark.parametrize(  # type: ignore[misc]
+    "n,blocks,expected_chunks",  # type: ignore[no-untyped-def]
+    [
+        (1, 1, [1]),
+        (2, 1, [2]),
+        (2, 2, [1] * 2),
+        (3, 1, [3]),
+        (3, 3, [1] * 3),
+        (3, 2, [2, 1]),
+        (7, 2, [4, 3]),
+        (7, 3, [3, 2, 2]),
+        (7, 7, [1] * 7),
+    ],
+)
+def test_split_array_chunks(n: int, blocks: int, expected_chunks: List[int]):
+    assert split_array_chunks(n, blocks) == expected_chunks
+
+
+def test_split_array_chunks__raise_on_blocks_gt_n():
```

Comment:
`split_array_chunks` could be tested with hypothesis as well - not that it must be, just saying :)

Reply:
I think some of the boundary conditions could be but I don't see a good way to test the core of what the function does from random inputs (other than basically copying the code over and re-applying it in the test).  Let me know if you're thinking of something different.

Reply:
@eric-czech 🤔 I think you could tests properties like:
 * size of the output equals the `blocks`
 * all elements are positive and monotonically increasing
 * sum equals to the `n`

Reply:
https://github.com/pystatgen/sgkit/pull/69/commits/bda112f96279b909983e3b9c7e47234d6ea9a973